### ISSUE TITLE
Alias changes value along with sortorder when changing list orders

### ISSUE THREAD

**Issue by DempseySbaiz** [KevinJump/uSync3.0#199]
Hey Kevin,

We've used uSync for a couple of months now to keep our development, staging and production environments in sync. It works pretty good, except for the known issue with the prevalues. You can't sort and rename a prevalue in one go, or else uSync has no info to rely on to make that change.

We also investigated why this is, and we also came to the same conclusion as you that the alias field of a prevalue changes along with the sortOrder when the field gets sorted. We find this very weird and confusing that an alias value would change when the sortOrder changes, we don't really see the difference between a sortOrder and an alias anymore.
Do you have any idea why Umbraco designed it to work this way, or do you know if it's a bug and if they are looking into it?

Also, if Umbraco were to change the behaviour of prevalues and keep the alias field the same, and only change the sortOrder field, would this be a fix for uSync to always have the right information to keep lists in sync, even if you rename and reorder in one go?

Sorry for the long post, and kudos for the great job on uSync.

Kind regards,
Dempsey

**Comment by KevinJump** [comment]
yeah - part of the issue is that diffrent data types work this out in diffrent ways - we are a bit dependent on the value resolvers in the data types 😦 . 

it is something we will put onto the next feature release thought to see if there is anything we can do to make it a bit better.
